src/scriptsys/lib_game.c

```diff
@@ -0,0 +1,131 @@
+#include "lua/lua.h"
+#include "lua/lualib.h"
+#include "lua/lauxlib.h"
+#include "scriptsys/script.h"
+#include "common/console.h"
+#include "containers/hash_set.h"
+#include "allocator/allocator.h"
+#include "lib_log.h"
+#include "lib_game.h"
+
+static HashSet update_handlers;
+
+static int func_start_update(lua_State* L)
+{
+	lua_pushvalue(L, 1);
+	i32 refId = luaL_ref(L, LUA_REGISTRYINDEX);
+	if (refId == LUA_REFNIL)
+	{
+		return 0;
+	}
+
+	Con_Logf(LogSev_Verbose, "script", "starting script activity #%i", refId);
+
+	if (lua_getfield(L, 1, "start") != LUA_TNIL)
+	{
+		lua_pushvalue(L, 1); // self arg
+		if (lua_pcall(L, 1, 0, 0) != LUA_OK)
+		{
+			Con_Logf(LogSev_Error, "script", lua_tostring(L, -1));
```

Comment:
Would be good to have ASSERT's as well, so that debugger catches these in a debug build.

Reply:
I guess we could make LogSev_Error raise a debugbreak actually. 
There are a few spots where this happens during map load due to missing textures, guess those should be warnings.